infrastructure/merklemountainrange/src/merklemountainrange.rs

```diff
@@ -63,17 +65,11 @@ where
         self.change_tracker.init(store_prefix, pruning_horizon)
     }
 
-    pub(crate) fn get_data_object(&self, hash: ObjectHash) -> Option<&MerkleObject<T>> {
-        self.data.get(&hash)
-    }
-
     /// This function returns a reference to the data stored in the mmr
     /// It will return none if the hash does not exist
-    pub fn get_object(&self, hash: &ObjectHash) -> Option<&T> {
+    pub fn get_object(&self, hash: &ObjectHashSlice) -> Option<&T> {
         let object = self.data.get(hash);
-        if object.is_none() {
-            return None;
-        };
+        object?;
```

Comment:
ln 71 and 72 can be combined ```let object = self.data.get(hash)?;```, then the ```unwrap``` can be removed from ln 73

Reply:
haha oops, good catch